app/src/modules/settings/routes/policies/add-new.vue

```diff
@@ -0,0 +1,61 @@
+<script setup lang="ts">
+import { useDialogRoute } from '@/composables/use-dialog-route';
+import { ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { useRouter } from 'vue-router';
+import { useSave } from './use-save.js';
+
+const { t } = useI18n();
+
+const router = useRouter();
+
+const isOpen = useDialogRoute();
+
+const name = ref<string | null>(null);
+const appAccess = ref(false);
+const adminAccess = ref(false);
+
+const { saving, save } = useSave({ name, appAccess, adminAccess });
+</script>
+
+<template>
+	<v-dialog :model-value="isOpen" persistent @esc="router.push('/settings/policies')">
+		<v-card>
+			<v-card-title>
+				{{ t('create_policy') }}
+			</v-card-title>
+			<v-card-text>
+				<div class="form-grid">
+					<div class="field full">
+						<v-input v-model="name" autofocus :placeholder="t('policy_name') + '...'" @keyup.enter="save" />
```

Comment:
This input is not limited same as the role name input should have a maximum length in sync with what the backend will accept (100 chars) 
```suggestion
						<v-input v-model="name" autofocus :placeholder="t('policy_name') + '...'" @keyup.enter="save"  maxLength="100 />
```